models/db.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type Requester interface {
```

Comment:
I'm not sure about this interface here. :/ It's bonded to Mongo and we aren't going to use any other database anyway.

Reply:
Well, you're probably right, I'd think of the different approach.